packages/client/components/AzureDevOpsScopingSearchFilterMenu.tsx

```diff
@@ -57,6 +72,43 @@ const AzureDevOpsScopingSearchFilterMenu = (props: Props) => {
         }
         onClick={toggleWIQL}
       />
+
+      {projects.length > 0 && <FilterLabel>Filter by project:</FilterLabel>}
```

Comment:
If I put line 76 in the return statement on line 96, we will get a `FilterLabel` multiple times as line 96 is called on each element being mapped on line 77. I think the label should stay where it is so that we don't get repeated `FilterLabels` in the drop down. If I have misunderstood your feedback, let me know. Thanks!